brax/physics/config.proto

```diff
@@ -61,6 +47,13 @@ message Collider {
   message Sphere {
     float radius = 1;
   }
+  // A height map aligned with the x-y plane
+  message HeightMap {
+    float size = 1;
+    int32 meshSize = 2;
```

Comment:
it seems your code assumes the mesh is square, which is fine by me.  If that's the case, can we remove this field, and then raise an exception if len(data) is not square?

Reply:
Yes, checking for squareness of len(data) and raising an exception is indeed the better choice.